src/sentry/static/sentry/app/views/groupEvents.jsx

```diff
@@ -160,7 +161,7 @@ const GroupEvents = React.createClass({
           {tagList.map((tag) => {
             return (
               <td key={tag.key}>
-                {tagMap[tag.key]}
+                {deviceNameMapper(tagMap[tag.key])}
```

Comment:
Can we not just do this for specifics tags? Seems a bit wasteful to do this to every tag.